### DISCUSSION TITLE
Sponsors pay out is stuck for 1 months

### DISCUSSION THREAD

**Discussion by LGouellec** [community/community#78662] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,

I received a Github sponsors one month ago, it still pending in my stripe express account. I don't understand how can I get paid to my bank account ?

It seems a similar issue than [54961](https://github.com/orgs/community/discussions/54961) or [60360](https://github.com/orgs/community/discussions/60360).

May someone help to sort this out?

**Comment by anjuan** [comment]
Hi @LGouellec, I'm the Engineering Manager for GitHub Sponsors. Can you confirm that your Stripe Account has been approved?

**Comment by LGouellec** [comment]
Hi @anjuan , 

My github sponsor is approved ..
<img width="777" alt="image" src="https://github.com/community/community/assets/3295357/8f344df2-8c8e-46a7-9fb9-90a5b8d26f4d">
and my stripe account too
<img width="757" alt="image" src="https://github.com/community/community/assets/3295357/678d8968-579f-46e2-8c1b-d27e504cd444">

**Comment by DHayes10** [comment, marked as answer]
Hi @LGouellec, 

Thank you for reaching out. 

For first payments, GitHub will remit to you all Sponsored Developer Payments 60 days after the first sponsorship is initiated, subject to the variables below. This probation period is applied to all Sponsored Developer Accounts and helps insure the integrity of the program.

You probation period is due to end on 2023-12-30. 

For more information, please visit - https://docs.github.com/en/site-policy/github-terms/github-sponsors-additional-terms

Best, 
GitHub Sponsors
